src/app/home/home.component.html

```diff
@@ -6,53 +6,33 @@
             </div>
             <div class="card-body">
                 <div class="mb-3">
-                    <input class="form-control" type="text" placeholder="O que está cozinhando hoje?" aria-label="default input example">
-                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3" placeholder="Deixe um comentário sobre a receita."></textarea>
+                    <input class="form-control" type="text" placeholder="O que está cozinhando hoje?"
+                        aria-label="default input example">
+                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"
+                        placeholder="Deixe um comentário sobre a receita."></textarea>
                     <input class="form-control" type="file" id="formFile">
                 </div>
                 <a href="#" class="btn btn-primary">Publicar</a>
```

Comment:
Essa parte aqui deveria ser um `form`, para que você ganhe algumas funcionalidades do HTML. Como,  por exemplo, quando o usuário apertar Enter, aí o form será submetido.

```html
                <form class="mb-3">
                    <input class="form-control" type="text" placeholder="O que está cozinhando hoje?" aria-label="default input example">
                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3" placeholder="Deixe um comentário sobre a receita."></textarea>
                    <input class="form-control" type="file" id="formFile">
                    <button type="submit" class="btn btn-primary">Publicar</button>
                </form>
```

No código acima, alterei o link para um `button` e adicionei o `type="submit"`, ou seja, você está dizendo que quando o botão for clicado, o form será submetido.

Ref de alguns outros `type` do `button`: https://www.w3schools.com/tags/att_button_type.asp

Reply:
Nossa, muito obrigado, vou alterar o codigo já! 

Mas minha dúvida é, isso não tem muito a ver com a PR q eu abri, eu devo abrir outra PR pra esse caso?

Reply:
Boa pergunta. Achei que tivesse adicionado os inputs nesse PR, mas como não fez isso, acredito que seria melhor alterar isso em outro PR, mesmo que seja um PR super pequeno, assim fica mais organizado. Da para criar uma issue para isso, para não esquecer de alterar.

Reply:
Beleza, muito obrigado! Vou fazer!